secret-contracts/calculator-with-permits/Cargo.toml

```diff
@@ -15,6 +15,10 @@ exclude = [
 [lib]
 crate-type = ["cdylib", "rlib"]
 
+[profile.dev]
+debug = true
+overflow-checks = true
+
 [profile.release]
```

Comment:
I think without it, it didn't raise an error when the operation caused an overflow. But now it's redundant because the ```checked_``` functions do it themselves